tests/NHapi.NUnit/Parser/LegacyPipeParserV21Tests.cs

```diff
@@ -54,6 +54,28 @@ public void TestObx5DataTypeIsSetFromObx2_AndAllDataTypesAreConstructable(Type e
             Assert.IsAssignableFrom(expectedObservationValueType, actualObservationValueInstance);
         }
 
+        [Test]
+        public void TestObx5DataTypeIsSetFromObx2_WhenObx2IsEmptyAndDefaultIsSet_DefaultTypeIsUsed()
+        {
+            var expectedObservationValueType = typeof(ST);
+
+            var message = "MSH|^~\\&|XPress Arrival||||200610120839||ORU^R01|EBzH1711114101206|P|2.1|||AL|||ASCII\r"
+                        + "PID|1||1711114||Appt^Test||19720501||||||||||||001020006\r"
+                        + "ORC|||||F\r"
+                        + "OBR|1|||ehipack^eHippa Acknowlegment|||200610120839|||||||||00002^eProvider^Electronic|||||||||F\r"
+                        + "OBX|1||||STValue||||||F\r";
```

Comment:
Can the generation of this message string be refactored into a function to be re-used across all the pipe parser tests? 
_e.g._
```csharp
string CreateInvalidOBX2Message(string version)
{
    return $"MSH|^~\\&|XPress Arrival||||200610120839||ORU^R01|EBzH1711114101206|P|{version}|||AL|||ASCII\r"
        + ...
}
```

Reply:
possibly... however, I think my preference would be to keep the tests isolated / decoupled so they don't rely on anything else, or any other test etc.

happy to discuss (and convinced) though.

Reply:
Makes sense to keep it as you have written :)